crates/fuel-core/src/schema/tx/assemble_tx.rs

```diff
@@ -105,6 +105,9 @@ impl<'a> AssembleArguments<'a> {
             return Ok(Vec::new());
         }
 
+        // TODO[RC]: Support partial flag in the AssembleTx transaction
+        let allow_partial = false;
```

Comment:
When we fetch required balances, it should be `false`, but when we fetch coins to cover the fee, it can be `true`

Reply:
Addressed in https://github.com/FuelLabs/fuel-core/pull/2912